NEW_PROJECT_APPLICATION.md

```diff
@@ -13,12 +13,20 @@ A rough description of the project in less than 100 words.
 Please refer to [the Cross Project Council's Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md).
 Please keep your answer to less than 250 words.
 
+## Impact and consumers of the project
+- What is the ecosystem impact of the project? 
+- Who are the biggest consumers of the project? 
+_if they are not JavaScript space, then will it ever expand to JavaScript space_.
+- What is the potential long term impact and adoption? 
+- How does it differ from other projects?
+
 ## List of all repos that are part of the project
 
 For each repository, please provide:
 
 - link to repo,
 - license information,
+- the IP belongs to,
```

Comment:
"who owns the IP?"

Reply:
I thought of this but none of other things were a question in that list.

Reply:
```suggestion
- who owns the IP
```
this is still more correctly phrased, and isn’t in question form

Reply:
Thanks @ljharb